Cypress Peer Dependency

will the Cypress Peer Dependency get update? i see its still on 4.5.0 which is very old.

We get:

```
▶ npm audit fix
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: @webpoker/tools@0.0.2
npm ERR! Found: cypress@5.2.0
npm ERR! node_modules/cypress
npm ERR!   dev cypress@"5.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer cypress@"^4.5.0" from cypress-plugin-snapshots@1.4.4
npm ERR! node_modules/cypress-plugin-snapshots
npm ERR!   dev cypress-plugin-snapshots@"^1.4.4" from the root project
```

Although the plugin clearly works with `cypress@5.2.0`. I would submit a PR to change the peer dependency to change the peer dependency to:

```
  "peerDependencies": {
    "cypress": ">=4.5.0"
  },
```

But I don't know if this plugin works with cypress versions higher than 5.2

--- a/package.json
+++ b/package.json
@@ -59,7 +59,7 @@
     "unidiff": "1.0.2"
   },
   "peerDependencies": {
-    "cypress": "^4.5.0"
+    "cypress": ">=4.5.0"
   },
   "devDependencies": {
     "eslint": "^5.14.1",

